### DISCUSSION TITLE
请教rec模型的训练方法

### DISCUSSION THREAD

**Discussion by lindadamama** [PaddlePaddle/PaddleOCR#12697] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/48863486/fb8a202e-ab42-4e94-abdc-c65dfceb9f68)
我想请教一下，我可以这样训练rec模型吗？我把每个训练集的图片都保持正向和旋转180度两种状态，目前训练集的验证结果比较令人满意，但是不确定可不可以这样做

**Comment by XiaokangLei** [comment]
> 请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
> 
> * 系统环境/System Environment：
> * 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
> * 运行指令/Command Code：
> * 完整报错/Complete Error Message：
> 
> 我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):
> 
> 请尽量不要包含图片在问题中/Please try to not include the image in the issue. ![image](https://private-user-images.githubusercontent.com/48863486/293312522-fb8a202e-ab42-4e94-abdc-c65dfceb9f68.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQwODk3MTUsIm5iZiI6MTcwNDA4OTQxNSwicGF0aCI6Ii80ODg2MzQ4Ni8yOTMzMTI1MjItZmI4YTIwMmUtYWI0Mi00ZTk0LWFiZGMtYzY1ZGZjZWI5ZjY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAxVDA2MTAxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwN2YzNjEyMTkwOGI5MGZkYzljOTJkMzdmZDdhOTA2MjMzNzE5MDRjMGRiZGNjYTdiOWJiM2M4NzVmNTM3YmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tTl-hEOYHDVV6PbgeUBjpMvhiQn7Cv8PLYN8B5Impro) 我想请教一下，我可以这样训练rec模型吗？我把每个训练集的图片都保持正向和旋转180度两种状态，目前训练集的验证结果比较令人满意，但是不确定可不可以这样做

不建议，个别正向和180度是不同字符的准确率会低，比如M倒过来是W，6倒过来是9
